web/main.c

```diff
@@ -0,0 +1,11 @@
+#include <emscripten/emscripten.h>
```

Comment:
these files in `web/` are all copypasta from that template repo, no need to review